### PR TITLE
New cross-entropy losses with temperature

### PR DESCRIPTION
**Pull request by cofri**
This PR introduces two new cross-entropy losses based on Keras standard losses and with a settable temperature for softmax:

- `TauSparseCategoricalCrossentropy` equivalent to Keras `SparseCategoricalCrossentropy`
- `TauBinaryCrossentropy` equivalent to Keras `BinaryCrossentropy`

### CODE REVIEW

**Review by franckma31 — APPROVED** [review]
looks fine for me

**Review by thib-s — APPROVED** [review]
looks good to me
